src/redux/reducers/gas-threshold-reducer.tsx

```diff
@@ -0,0 +1,58 @@
+import { IReduxAction } from 'redux/interfaces/IReduxAction';
+import { updateObject, createReducer } from 'redux/reducers/reducer-utilities';
+import { ProcessState } from 'models/ProcessState';
+import { ReduxGasThresholdType as t } from 'redux/redux-types/gas-threshold-type';
+
+export interface IGasThresholdInfo {
+  chainId: number;
+  threshold: number;
+  networkName: string | null;
+}
+
+export interface IGasThresholds {
+  [key: string]: IGasThresholdInfo;
+}
+
+export interface IReduxGasThresholdState {
+  fetchState?: ProcessState;
+  gasThresholds?: IGasThresholds;
+  error?: string | null;
+}
+
+const initialState: IReduxGasThresholdState = {
+  fetchState: ProcessState.READY,
+  gasThresholds: null,
+  error: null,
+};
+
+const fetchGasThresholds = (state: IReduxGasThresholdState, actions: IReduxAction) => {
+  return updateObject(state, { fetchState: ProcessState.PROCESSING });
+};
+
+export interface IReduxGasThresholdsFetched extends IReduxAction {
+  type: t.GAS_THRESHOLDS_FETCHED;
+  payload: {
+    gasThresholds?: IGasThresholds;
+  };
+}
+
+const gasThresholdsFetched = (state: IReduxGasThresholdState, actions: IReduxGasThresholdsFetched) => {
+  return updateObject(state, {
+    fetchState: ProcessState.HANDLED,
+    gasThresholds: actions.payload.gasThresholds || null,
+  });
+};
+
+const fetchGasThresholdsError = (state: IReduxGasThresholdState, actions: IReduxAction) => {
+  return updateObject(state, { fetchState: ProcessState.HANDLED, error: actions.payload.error?.toString() || null });
+};
```

Comment:
As above re: inlining...

Reply:
Haha yeah sorry, I use a standard template I made for these redux functions so I overlooked these.

Reply:
It's all good!